clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/model/GoogleServerGroup.groovy

```diff
@@ -110,8 +111,12 @@ class GoogleServerGroup {
     Set<String> getLoadBalancers() {
       Set<String> loadBalancerNames = []
       def asg = GoogleServerGroup.this.asg
-      if (asg && asg.containsKey("loadBalancerNames")) {
-        loadBalancerNames = (Set<String>) asg.loadBalancerNames
+      // TODO(jacobkiefer): This is fragile.
+      if (asg && asg.containsKey("regionalLBNames")) {
```

Comment:
I'm not a fan of these magic strings - Pull these keys into public static constants (within the View class), and use them here, and in the `Google[Zonal|Regional]ServerGroupCachingAgent`.

I would also expand LB - `regionalLoadBalancerNames`.